dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jcaSecurity-1.0.feature

```diff
@@ -1,7 +1,7 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=com.ibm.websphere.appserver.jcaSecurity-1.0
 -features=com.ibm.websphere.appserver.containerServices-1.0
--bundles=com.ibm.ws.security.jca, \
+-bundles=\
```

Comment:
This corrects a mistake in a previous delivery.  The inclusion of this bundle vs the jakarta version of it is done by the autofeature, and should not be here in the main jcaSecurity feature.